src/js/out/modules/node/child_process.js

```diff
@@ -705,7 +745,6 @@ var child_process_default = {
 export {
   spawnSync,
   spawn,
-  fork,
```

Comment:
How come the named export is removed?

Reply:
I lost the export on the function, i fixed it